README.md

```diff
@@ -236,17 +236,19 @@ export default Component;
 
 **The following environment variables are embedded during the build time.**
 
-* `process.env.RAZZLE_PUBLIC_DIR`: Path to the public directory.
+* `process.env.RAZZLE_PUBLIC_DIR`: Absolute path to the public directory in the server's filesystem.
```

Comment:
Pretty sure it can be relative.

Reply:
It technically can, but the default is a path.resolve-generated absolute path. I figure it's better that it predictably be an absolute path, and anyone putting in a relative path can do so at their own risk. I changed the wording in the doc because "Path" alone didn't tell me to expect the default to be "/public" or "/full/path/to/public/"